tests/Vote.Monitor.Api.IntegrationTests/Country/ListEndpointTests.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Vote.Monitor.Api.Feature.Country;
+
+namespace Vote.Monitor.Api.IntegrationTests.Country;
+public class ListEndpointTests:IClassFixture<HttpServerFixture>
+
+{
+    public HttpServerFixture Fixture { get; }
+
+    public ListEndpointTests(HttpServerFixture fixture, ITestOutputHelper outputHelper)
+    {
+        Fixture = fixture;
+        Fixture.OutputHelper = outputHelper;
+    }
+
+    [Fact]
+    public async Task Should_ReturnAllCountries()
+    {
+        // Arrange &c Act
+        var (response, result) = await Fixture.PlatformAdmin.GETAsync<Vote.Monitor.Api.Feature.Country.List.Endpoint, List<CountryModel>>();
+        
+        // Assert
+        response.StatusCode.Should().Be(HttpStatusCode.OK);
+        result.Count.Should().Be(249);
+    }
+
+    [Theory]
```

Comment:
Having it as a test theory it is overkill for an integration test. We should move these sanity checks inside the test method and check that indeed we have in the response this small sample of countries.

Reply:
solved